src/Uno.UI/UI/Xaml/Media/GradientBrush.Android.cs

```diff
@@ -13,6 +14,8 @@ protected override Paint GetPaintInner(Rect destinationRect)
 			return paint;
 		}
 
+		Shader IGradientBrush.GetShader(Rect destinationRect) => GetShader(destinationRect);
+
```

Comment:
Why an explicit interface implementation since both interface `IGradientBrush`  and the `GetLayer` method are `internal` ?
```suggestion
```